### DISCUSSION TITLE
Может туториал посмотреть?

### DISCUSSION THREAD

**Discussion by ghost** [extremecodetv/ExtremeCodeOS#185] [Ideas]
Немцы умеют строить все и главное качественно. Вот туториал от одного из них, тупо следуйте ему.

линк: https://youtu.be/1rnA6wpF0o4

Не благодарите ♥️

**Comment by hawr-desk** [comment]
Мне кажется или в данном видео человек рассказывает про создание ядра ОС, а не дистрибутива?

**Reply by ghost** [reply]
Там про создание ОС, в него входит все начиная с нуля.

**Comment by ghost** [comment]
Нам не надо уториал у нас новая OS и там все должно быть по новому!
